### DISCUSSION TITLE
Why did the software installation fail

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#1340] [Questions & Problems]
![6ced3623407c6f9a7a493b84afe5792](https://user-images.githubusercontent.com/81560924/195848282-c5ae4d2c-f0a3-4a0b-9b8e-49ebd0fbb9ed.jpg)
When I install the software, this window pops up. 
How can I solve this problem?

**Comment by Yuri05** [comment]
Seems your Windows Installation is corrupted.
Try to fix as described here: https://support.microsoft.com/en-us/sbs/windows/fix-windows-update-errors-18b693b5-7818-5825-8a7e-2a4a37d6d787
